src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -120,7 +122,7 @@ Uri GetPackageContentAddress()
                 {
                     string id = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Id)).FirstOrDefault().Object.ToString().ToLowerInvariant();
                     string version = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Version)).FirstOrDefault().Object.ToString().ToLowerInvariant();
-                    string path = string.Format("packages/{0}.{1}.nupkg", id.ToLowerInvariant(), version.ToLowerInvariant());
+                    string path = GetPackagePath(id, version);
```

Comment:
I used a Function, because I need it to overridable. So it's not shorthand, it Fucntion<string, string, string>

---
In reply to: [57124311](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57124311) [](ancestors = 57124311)